build/packages.json

```diff
@@ -3,6 +3,7 @@
     "requiredNPMVersion": 7,
     "packages": [
         "build",
+        "platform-utils/endpoints-generator",
```

Comment:
Wouldn't this just be a `dev-util`? Not terribly fond of new directories...

Reply:
It's not for devs, it's necessary for the api to work. dev-utils is like nice to have for devs. So I feel it's quite a different use case

Reply:
Yet I'm happy about suggestions, agree that this wouldn't need a new dir, while I couldn't figure out which one it would fit in

Reply:
feature-utils then? Or maybe dev-utils/platform?

Reply:
It's not part of features and as said I don't think it makes sense to be in dev utils. In the end it's just an artificial build step for the secrets

Reply:
Not finding the right place for it might be a red flag... Maybe you should just use the variables directly in the rollup script. I don't know, but as said elsewhere, the generation of a file with secrets itself is filled with danger...

Reply:
Before the secrets were in the feature file as well. The safety of the secrets would not decrease at all. This way would not work for external devs though as it is purely internal so we can't continue with this for long. Increasing the safety of these secrets should be a different PR and not block this one right?

Reply:
OK, they're going to end up as an unencrypted string in the app no matter what, and it can possibly be extracted. Security through obscurity does not work. But still generating a file that holds secret in a space that can be committed by error is an additional degree of possible problems.
So there are two issues here that are blockers
1. The directory where this is placed. New directories imply not only adding it to the manifest, but also additional configuration in the workflow paths and so on. The current layout was decided after no small amount of thinking, so adding a new top-level one should receive at least the same amount. I would settle for almost anything that's a second-level directory such as platform/utils (which semantically is not so different).
2. The existence and placement of the file. For me, the ideal would be to include somehow the secrets in the gradle/XCode build tools so that they're "seen" only at the point of compilation; once it's there, it's going to be seen if you look hard enough. That's a considerable amount of work, though. Second best: just place the temporary file _outside_ the repository. You can safely use ".." (because it's going to be something like `/home/user` in CI, and whatever you have locally, point is, it's not going to be a directory), or even /tmp. For an additional degree of security, you can delete it as soon as you create it. Additional security, have it reside in a random file name. Not any of that is really needed, but simply using a non-repo directory for it would be enough to unblock this.

Reply:
1. Sure lets go (already done)

2. I don't understand whats going on here. The secrets will only be in the automated build done by github. No one else will have them in their build. Where would the file go when github builds that and it's outside? Sorry I can't follow your thoughts here

Reply:
But does that work for the github automated build? The private ones as said will not contain the secrets anyway

Reply:
And is it really necessary then?

Reply:
Yep. Github automation should have `/home/github` that's always writable by the actions, and also /tmp. Checkouts happen at `/home/github/repo-name` so going up from the checkout repo will always be writable locally or in CI.